src/Components/LandingPage/LandingPage.tsx

```diff
@@ -62,11 +64,11 @@ const LandingPage = ({ handleCheckboxChange }: LandingPageProps) => {
 
     if (formData.agreeToTermsOfService && formData.is13OrOlder) {
       if (uuid) {
-        navigate(`/${uuid}/step-2`);
+        navigate(`/${uuid}/step-3`);
       } else {
         const response = await createScreen(formData);
         screenDoneLoading();
-        navigate(`/${response.uuid}/step-2`);
+        navigate(`/${response.uuid}/step-3`);
       }
     }
   };
```

Comment:
I am going back and forth about whether this should be done after the language question or after this question. What do you think.